docs/advanced-solidus/returns.mdx

```diff
@@ -0,0 +1,117 @@
+via form or email. At that moment, the customer service can create an RMA
+(Return Merchandise Authorization) with a unique number that can be used as a
+reference for any further communication.
+
+In Solidus, an RMA is represented by the
+[`Spree::ReturnAuthorization`][return-authorization] model. It contains a
+number and other information like the stock location where items should be
+delivered or a memo for a free-text description.
+
+A `Spree::ReturnAuthorization` has an authorized state on creation, but it can
+be canceled if desired. We must note that, at this point, we're still dealing
+with user petitions, not actual returns. A store can authorize a request and
+keep processing it, regardless of its final result (e.g., not ending up
+accepting a return), or it can cancel it for being invalid and not deserving
+further consideration.
+
+## Return items
+
+An RMA must reference one or more of the products a customer purchased. For
+instance, a user bought five cups, but one of them arrived damaged. Depending
+on the store policy, they might ask for a reimbursement or an exchange for
+another cup. At a later stage, the request will be approved, edited or denied.
+
+In Solidus, each item is represented through a
+[`Spree::ReturnItem`][return-item] record. Notice that a `Spree::ReturnItem`
+instance is associated with a [`Spree::InventoryUnit`][inventory-unit]. That's
+the most concrete level of product identification in orders. I.e., orders can
+have several line items for different variants, and each line item may contain
+one or more. The inventory unit maps one to one to a physical (or digital, for
+what it's worth) object, such as a broken cup.
+
+A `Spree::ReturnItem` contains information about the requested reimbursement
+amount and how it should be processed (e.g., refund or store credit) or another
+unit with which the store should replace it.
+
+Finally, two status fields govern the lifecycle of an item for return:
+
+- The **acceptance status** controls the eligibility of the return per-se. Initially
+_pending_, it should end up _accepted_ in case of going forward.
+
+- The **reception status** marks whether the store has already received the
+returned package. In a standard scenario, it should transition from _awaiting_
+to _received_.
+
+## Customer return
+
+Finally, after the return authorization has moved forward, it'll materialize in
+a customer return. At that point, a store will already have the returned items
+back. It might be that the customer support department rejected some of the
+requests or accepted all of them.
+
+In Solidus, that's represented by a [`Spree::CustomerReturn`][customer-return]
+record. Similar that RMAs, it's a simple model. It only contains an
+identification number and a stock location that is as a final override of the
+RMA's proposal. Customer returns are flexible and not tied to a single return
+authorization. The important association here is that a `Spree::ReturnItem`
```

Comment:
I rephrased it to:

>  Customer returns are flexible and not tied to a single return
authorization. Because of that, `Spree::ReturnItem` belongs not only to
`Spree::ReturnAuthorization` but also to `Spree::CustomerReturn`.

WDYT?